IFC4x3/Properties/d/DipDirection_17WEfvK3L9Ehv8Rdn5AK_c/Documentation.md

```diff
@@ -0,0 +1 @@
+The geographic  direction of a line created by the intersection of a plane and the horizontal plane. If non specific convention is used, this angular value is in the range 0 to 18 degrees. In this case, there is an ambiguity on the dip orientation. For example, a plane with an orientation of 90 degrees from the nord could either have a dip direction to the North or to the South. Thus, to avoid this ambiguity, a strike value is generally completed with an indication of the dip orientation (Quadrant).
```

Comment:
```suggestion
The geographic direction of a line created by the intersection of a plane and the horizontal plane. If non specific convention is used, this angular value is in the range 0 to 180 degrees. In this case, there is an ambiguity on the dip orientation. For example, a plane with an orientation of 90 degrees from the north could either have a dip direction to the North or to the South. Thus, to avoid this ambiguity, a strike value is generally completed with an indication of the dip orientation (Quadrant).
```